agimus_sot_msgs/Makefile

```diff
@@ -0,0 +1,21 @@
+# robotpkg Makefile for:	wip/agimus_sot_msgs
+# Created:			Long Ha Thuc on Tue, 18 Aug 2020
+#
+
+ORG=			agimus
+NAME=			agimus_sot_msgs
+VERSION=		1.0.0
+DISTNAME=		${NAME}-${VERSION}
+
+MASTER_SITES?=      ${MASTER_SITE_GITHUB:=${ORG}/${NAME}}/releases/download/v${VERSION}/
+MASTER_REPOSITORY?=	${MASTER_REPOSITORY_GITHUB}${ORG}/${NAME}.git
+LICENSE=		2-clause-bsd
+
+CATEGORIES=		wip
+COMMENT=		Messages and services for the agimus_sot package
+
+include ../../devel/ros-catkin/depend.mk
+include ../../mk/language/c.mk
+include ../../mk/language/c++.mk
+include ../../mk/robotpkg.mk
+include ../../devel/jrl-cmakemodules/Makefile.common
```

Comment:
Could you put this as the first include ? I'm not sure, but I think this defines important constants for the other includes.